### DISCUSSION TITLE
Do we have any plan to support the Chart element?

### DISCUSSION THREAD

**Discussion by PublicWorld** [saadeghi/daisyui#2200] [Ideas / feature requests]
Hi, I'd love to use the daisyui, but I want to know do we have any plan support this?
such as this screenshot, thanks~
<img width="323" alt="image" src="https://github.com/saadeghi/daisyui/assets/12987627/47ce7cd7-3ddd-4195-bf47-0973fcb8b9ae">
<img width="287" alt="image" src="https://github.com/saadeghi/daisyui/assets/12987627/1fa70a9b-ed8b-4a55-be7e-724119bc0676">

**Comment by saadeghi** [comment]
daisyUI (as a plugin for Tailwind CSS) is CSS-only. It doesn't inject any JS code to your app.

The first screenshot is not possible to do with CSS (technically it is possible to draw it using CSS as a static picture, but that's not useful to use in a real project. In real life, you would provide an array of numbers and you would expect a chart in the output.)

Currently the best way to do that is using JS chart libraries which draw lines on a canvas element.

There are many chart libraries for pure JS and for each JS framework that you can use. Using canvas element, they can do things that CSS cannot do. And they provide many options and configs. It's totally fine to use third-party JS library alongside daisyUI.


You can make the second screenshot using [radial progress](https://daisyui.com/components/radial-progress/)

Let me know if you have any questions
